app/views/pages/_placeholder.html.erb

```diff
@@ -1,13 +1,9 @@
 <% title "#{title} #{community_qualified_name}" %>
 
-<div class="blank-space"></div>
-<div class="container article">
-  <div class="title">
-    <h1><%= title %></h1>
-  </div>
-  <div class="body">
-    <p>
-      The production <%= title %> page is a page generated at <a href="/internal/pages">/internal/pages</a>! This one is just a placeholder.
-    </p>
+<div class="crayons-layout crayons-layout--limited">
+  <div class="crayons-card text-styles text-padding">
+    <h1 class="fs-3xl s:fs-4xl l:fs-5xl fw-bold s:fw-heavy lh-tight mb-4 mt-0"><%= title %></h1>
```

Comment:
I think I'll turn page title into its own component in next PR - I've been using combination of these classes often enough to make it a component I think.